meilisearch/task.py

```diff
@@ -67,6 +68,37 @@ def get_task(config: Config, uid: int) -> dict[str, Any]:
     return http.get(f"{config.paths.task}/{uid}")
 
 
+def cancel_tasks(config: Config, parameters: dict[str, Any] | None = None) -> TaskInfo:
+    """Cancel a list of enqueued or processing tasks.
+
+    Parameters
+    ----------
+    config:
+        Config object containing permission and location of Meilisearch.
+    parameters (optional):
+        parameters accepted by the cancel tasks route:https://docs.meilisearch.com/reference/api/tasks.html#cancel-task.
```

Comment:
Is `route` supposed to be here?

```suggestion
        parameters accepted by the cancel tasks https://docs.meilisearch.com/reference/api/tasks.html#cancel-task.
```

Reply:
No! Thanks